lua/neogit/config.lua

```diff
@@ -59,12 +59,10 @@ M.values = {
     item = { ">", "v" },
     section = { ">", "v" },
   },
-  integrations = setmetatable({}, {
-    __index = function(_, key)
-      local ok, value = pcall(require, key)
-      return ok and value or false
-    end,
-  }),
+  integrations = {
```

Comment:
The previous auto integration relies on the metatable being left intact and would break if the table was replaced. Additionally, this makes the config data only and does not muddle printing or probing of the config with runtime determined fields.

This functionality is now *ensured* by the `check_integration` function